Parallelise Troubleshooting section in Admin Guide (#450)
SUSE/doc-susemanager: modules/administration/pages/tshoot-packages.adoc

[[troubleshooting-packages]]
= Troubleshooting Package Inconsistencies

////
PUT THIS COMMENT AT THE TOP OF TROUBLESHOOTING SECTIONS

Troubleshooting format:

One sentence each:
Cause: What created the problem?
Consequence: What does the user see when this happens?
Fix: What can the user do to fix this problem?
Result: What happens after the user has completed the fix?

If more detailed instructions are required, put them in a "Resolving" procedure:
.Procedure: Resolving Widget Wobbles
. First step
. Another step
. Last step
////

Packages can sometimes be locked or taskomatic can experience problems, which creates problems with metadata regeneration.
When this occurs, package updates will be available in the {webui}, but will not appear on the client, and attempts to update the client will fail.
To correct this, determine if any processes are running, or if a crash could have occured.
Check package locks and exclude lists to determine if packages are locked or excluded on the client.

> Once you have located the problematic process, the metadata can be regenerated and synchronization occurs as expected.

Reviewer: ```suggestion
When you have located the problematic process, the metadata can be regenerated and synchronization occurs as expected.
```

Reviewer 2: :+1:

.Procedure: Resolving Package Inconsisties

> . On the client, check the [path]``/var/log/rhn/rhn_taskomatic_daemon.log`` configuratiopn file to determine if any processes are still running, or an exception which could indicate a crash.

Reviewer: ```suggestion
. On the client, check the [path]``/var/log/rhn/rhn_taskomatic_daemon.log`` file to determine if any processes are still running or a crash occured.
```
the log file is not a config file ;) and the rest could be simplified, IMO.

Reviewer 3: And this logfile does not exist on the client, but only on the server.

Reviewer 2: :+1:
